wp-content/mu-plugins/postcrafter-yoast-integration/includes/class-rest-api-handler.php

```
@@ -85,7 +217,234 @@ public function check_permissions($request) {
+        
+        $primary_plugin = $this->detector->get_primary_plugin();
+        $update_results = array();
+        
+        // Get field values from request
+        $meta_title = $request->get_param('meta_title');
+        $meta_description = $request->get_param('meta_description');
+        $focus_keywords = $request->get_param('focus_keywords');
+        $canonical_url = $request->get_param('canonical_url');
+        $robots_noindex = $request->get_param('robots_noindex');
+        $robots_nofollow = $request->get_param('robots_nofollow');
+        $opengraph_title = $request->get_param('opengraph_title');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$opengraph_title'. (Unused Code Rules)

(UnusedLocalVariable)